tests/serverspec/default.yml

```diff
@@ -67,8 +67,8 @@
   roles:
     - ansible-role-virtual-interface
   vars:
-    virtual_interface:
-      gre0:
+    virtual_interfaces:
```

Comment:
backward compatibility can't be preserved by keeping variable name, because `dict` and `list` are not compatible anyway